gadget/layout.go

```diff
@@ -310,6 +314,11 @@ func resolveVolumeContent(gadgetRootDir, kernelRootDir string, kernelInfo *kerne
 // content and populates VolumeContent.resolvedSource with absolute
```

Comment:
I didn't get a chance to point this out in the previous pr, but this function doesn't actually populate VolumeContent.resolvedSource, it is used that way in a different function, but this doc-comment reads weird since it doesn't actually take a VolumeContent at all in it's args/return values

Reply:
indeed this comment is wrong now, also maybe resolveVolumeContent merits a comment too

Reply:
I will update in a followup